libio/string/pcre_compile.c

```diff
@@ -3389,7 +3389,7 @@ while ((c = *(++ptr)) != 0)
     case '+':            /* those are handled separately */
     case '?':
     length++;
-    goto POSESSIVE;      /* A few lines below */
+    goto POSSESSIVE;      /* A few lines below */
```

Comment:
I could probably adjust the whitespace here (and below).

Not sure how attached to this file things are -- also happy to drop entirely / skip files.